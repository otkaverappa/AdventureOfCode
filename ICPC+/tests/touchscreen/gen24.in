19
zpqwzyxqjxxdgwfsmchskkzv 10
kystjfjkwzgwadmlcjokhtet
xlniymdkkydvdofzqoxqtjby
uskelpxlcltazwkmwqjzerbu
hymckqcginkvckggvbjwavkw
lwyqnzmwxzahrdobqawukfah
gjegfodsmbkzaxxzyyjpbzqt
zonkvprczxigonaaqlasizsi
xdzacrtbhgldwffxcpfqdftv
qtoauiisljsnamqiuelqjspn
hugmazkquytoidivnbippyxl
ckcnctbmohaogkgakzqsdzns 10
axhryhccrgqwatkqakeixmkh
oactzplbpsunbzpvfhrhadxc
pdmmqyuebwxankeceargzjbh
quotxmxmpmzhkvnltmogwuld
uejunmdggrbffatxmuexpukk
gyrfseipkrlyfognfjsnjlkw
gqvxkfirgbyahgprzcreqxry
gjlswxoenlcasmtynrzuxqlw
tdclctjlewdcvthjgllyxexm
vyisqvqjasxdniqtfvvcoenw
pyuocrczrktigjtgdqlszdlg 10
yjiopwkhwfvbyaaskuasfvbl
mmdnsrtqcdfrbryawwbxwbpi
vrbdmcoyqtnimibomigpzgpy
cqvyumhrdkupmkqeedpsmqha
yppzvhxazuatgjnjtjbhurma
wbsitbktscsqjsqkmqgvatgt
chdyapyzstjnuugnwaffsvrh
nxcnqijvsmtudutxndkjzqyv
sdalytumrwbjgmeaayxdusah
xmqwerrxutkvnehecknkxunz
ukdoefvcsnyyeqvzlhuyldep 10
puaoopoizrwfythsggrnyooj
wkkhnozeiztyohgpaeuyzeti
mmvkajvwthgiyfoighhwopnq
wjrvnkdzwalwlhvgqbooidxq
mgmdvztrikpwwuxsvjrgsoni
rdxziwpvcdaxctrmgikccixz
rogjcvrvaqulmliooiosbhgj
prmrbjrszzddwwywpuhbfrsv
zgpdpvmemaxnjqhkpmpnjnmy
jvbontlmbarqwewkeuaqmhcc
usrfhdfrzgfmctadtswryucc 10
ocublxfhrymycrpdyxraqrfl
lddjyhoojkrwjweavqzzkoci
lvkbmrpaxslvbzjkmbivzmww
fvxplzzxulajcplbjwwkyiwk
jffktbhyyenkgohczjnebyfk
xcxvktfvakhuooumvkwdyfhy
pxeqvjcunzpzvuuvhcrvsohp
yguwncwezawwkzrzagzvdvtk
xmfsaojawexkgvohwldimwhm
dgkgcdsbrxvsnfuklrutnkal
vfvhccwhlgpplhscgqwtxrfi 10
kzbzmdkjihrkmptxviogpikx
yhrvawfkyikmlwvugmgsdapy
ifgantzocqlcpropbybnxwjd
lpvormpatxcgsbwuuiwlblcc
ldpkbypoqkfhywjrtnampwgl
gfyhqamdfbogcdvuqabqwlkr
akfriocrtaylakqhoeosklmc
npsjcccdokuzyzqtzqgcdwjt
dzlpkzrxqlitqnwgxsfvrxqt
owxrsgkxixmswfsmrahjpfpm
yxjruzkivhbqqnnykcsihkxa 10
nejckzpkyybsamdvvenlsdle
fenmoknboygaxxkxvoqxcvvx
ziltlwysanfrxuunsaqsxcpv
qhsscnseydzlzxecmltlfnzz
qrrntjimsdgvsyzqbycdygfm
rayynzxdqrtlcdxugepzcqrg
rtlrbqdusdsfcslslggnjdhs
jztmrkuigfbhxfepkwwnqjhb
pnobrytaxmooykzgpcqohufu
qchhnokfczgvxzxwnomnynuq
qkezektwoafdpqitpoqoppke 10
dyscmmucwabamvwcvcgmuohj
fzzuplbsjtwvfqzbtbbfyziv
dokxetijthgishceabbhtbim
elsemazqqjpudygyhoicxmgz
njghkovqbnunpufhfvckvkic
ysevfmuswcbisxztmtiepqlx
nphizskzmrwrdsmzunimmjhb
esfuistxkcflwrljkjaocopy
cznqiurnoyjyrevbgcodwzog
lqwnfonhnazyxrnlpykichlk
kapibdomwnbbbpkqpjooadas 10
bmafvoqhohprnggjthkwywmq
fdeigeckseposhvirlaergnm
pziovueczkkgrosjthzlrvwi
iyobgbnvaxlxtrztcmztatet
cehtbddlbrmhubewaqttjtml
fngfglbjpieslkdodrxysbus
tqldjaqqnxxwkyhbilttvwjz
ngzhkvbflokwodmeamckmjnv
ugqsecttksaundczrovirkot
yqdkzshvyyneagxkyzhoclnw
zkerwskukqhjioejounqanbb 10
mkprvenwotpnnchxuqiefont
idmlqpoezfxwlkubfkosmwrg
mbmtqapafbnxqdbribntlhvt
tlnhjfpwgcrycgbjjogzujse
kfaypxrkiesslhqulhupqvyz
mgbgqtkcbkashtesaykngaht
hejzzhznoctgxgkasnucgzui
xewfefznjinlroyhssppaass
pmuvnqgluesajtovbdgvreel
ywcawutljpiwgqjcucdgvtdz
wjwppaaqwcquzlhidselionf 10
ssnnlqmhbkyslaijfbeemnor
fscqiqvaiiqwyefbpetagcll
fpptfflkynaifxjqfbmfftgu
xaweeirjzjceoppodrxkpicw
jqeokmljohpuphdpqitgxkvc
csntcrrnixbsmodawtxlcaau
kwakgvmkpbfrvzedwixiwalu
tigvkirxgshnnvzfzfyugfxc
punnwyisiqnsyhpfzyuouvtt
cunizmlqiadealwibmactsjs
qfimaeffyunziapqasxagtij 10
hllcfwwxbfkejrjhowgwzxpz
pmcxhmhrzstepqcsvmwgdhpr
gxqfufgltijcxqtwipbzffsc
rqiwaxqgwgnqntbiekkbaezl
tamafgcyxmvzlnfhvszilaqr
ncunguabundcthastxrgkwph
qopephvekqrskuwghzkchnvc
mmlzlagbpwhgfdkpvdkhzgpi
halongtzueyhghlxdsfkxras
xmbwsrfztsqgylhuricxppvu
haefugxrsbnmsuolmgtmrcji 10
knhbccxjfeozmoshpftjajuo
ppbirksexahzefkjlzkxpceg
kapmlladadlrpgvogdnkkywv
ziuqnbwxdllownrwrfpilnxt
smdcmbzlkvdxwzwzkjqiwjgp
qwydlxwdjcfwdgjnbnmamkbz
trjsdqhtohxzeveoxlmbrvqu
ifwxpywlrfduxkqnspoyksok
daluwdrgindylbjfjmzgyrws
gkrtdfdigqdetwkejqcxrnca
bdjbvhudrlywsdfatkepgrtr 10
hxqymubnymovtkymvzlqcqqw
cxnkogdvfuvroygomwjfhkue
jfulxnhbkulycovhkqabrgpd
dzlkjfosmldlynmijziopdxa
tybmeqqjrbvaikuuxyfvntdw
snkjqkllimyofoaypvzxhvte
vazivehprtyjdkvnytcfjcea
xfzhaslwungptpenjcwoorcn
meuviyxhdwogqaekpmbicfvl
kuzanbnahixpivxotluknawe
mynqfkbredsughuoptfaqeok 10
skwhmumauzralsrsxlmetiuk
bbkugagakchyytztuqvgioai
amotwidxlottqztdbbbawavr
qqzbezlenzzjjdiutcojbjmd
mqfjqcajsbmyaxfnweyfhgbc
irmkbapnqwwjayuszgsbdxoz
coglvhodzcpcegpveogemawo
grpjogksusepzuvawmcatuxx
idcufakltawigicaagsccpcy
dhzwbywjcafjcqxwsvgyekag
ruktjpunwvlztjkxlrhohemz 10
zuzfeaoxuytgnnvligmdswbd
oitxohzpbyxialhxlafbpaoz
ibdaxgenqxlehmvkkssnfamt
crusrktzlwclegaufnymzuxm
orzwrlpwelqvvjwjiyumgwhm
khajdzvsruokhfglqwilihuq
hqeonncxvegaeeuvakhipqvf
mgrwpnowetmtiqqdvygbcayd
lfnavkikqbifoxeusqnajfgg
emhiofnzmacknkwdnelbbryv
hlwsteyxqfhemvgbxilkujqh 10
ndkpuimdwiwpmuoecxlqsrrs
beexnwebzqsvzezxpvmdrciv
zvmuofmrksozpuaokumjblgq
iutbwdxyyjsoogfaytanncey
xqjydsqlmlmlpjjpudeljllj
flzuodunvelaydlmozzflixf
ndqwoeitphqdmkqhqdhoiubz
vaeglboaegzvkhqcogharajh
fsyombnjbtsoxgpdooybvpdl
vmlomuysmwgaxvjaobrljgpa
wnbtegecspqflqzxmhzlcios 10
mfexlvxjkacpggtbvjiiahfo
oecsoqkaxqyknxtyyvpgbihy
tpgvynmmtogjgrkdhknxhivh
dmogwvgrnonldbawpgfxxrch
dpenxbucpikngqetgtejughl
nokmimvlbbyaevetdoijenem
ilvfrcqgtbtbnqmrsntwiysn
maztpdfyqddkhvqawlbmcqfu
dysnwkalnbecflaxofhwbywz
jannqsjvtcirokcbmigtuiqk
oygryfqjfgxxyisrkclbmney 10
wmrsviclilcgqupxaowbxouj
qfmctrdrdujaenlmynuqikpi
aljxbejtlvyeobxsxgscweow
tlnbxcmxqxxrdglpdlvsmtmj
belxkcuenjhkntjdtgxwoils
vhmhcysdcedpixvxgcjvyszr
bwnpgbjbkvimtapygsnqrknz
mwvmrudsstjyutzeqjsklhku
czktjavvyqirmnjegvecofgg
qyqbicxkbkfkmcikuscgfnmo
