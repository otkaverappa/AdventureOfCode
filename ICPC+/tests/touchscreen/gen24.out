xdzacrtbhgldwffxcpfqdftv 81
gjegfodsmbkzaxxzyyjpbzqt 86
xlniymdkkydvdofzqoxqtjby 94
hugmazkquytoidivnbippyxl 97
lwyqnzmwxzahrdobqawukfah 97
zonkvprczxigonaaqlasizsi 99
hymckqcginkvckggvbjwavkw 100
qtoauiisljsnamqiuelqjspn 101
kystjfjkwzgwadmlcjokhtet 103
uskelpxlcltazwkmwqjzerbu 117
gyrfseipkrlyfognfjsnjlkw 86
quotxmxmpmzhkvnltmogwuld 87
vyisqvqjasxdniqtfvvcoenw 91
pdmmqyuebwxankeceargzjbh 92
gjlswxoenlcasmtynrzuxqlw 93
tdclctjlewdcvthjgllyxexm 95
axhryhccrgqwatkqakeixmkh 96
oactzplbpsunbzpvfhrhadxc 96
gqvxkfirgbyahgprzcreqxry 98
uejunmdggrbffatxmuexpukk 101
mmdnsrtqcdfrbryawwbxwbpi 71
chdyapyzstjnuugnwaffsvrh 74
nxcnqijvsmtudutxndkjzqyv 74
cqvyumhrdkupmkqeedpsmqha 81
yppzvhxazuatgjnjtjbhurma 83
sdalytumrwbjgmeaayxdusah 91
wbsitbktscsqjsqkmqgvatgt 92
yjiopwkhwfvbyaaskuasfvbl 94
xmqwerrxutkvnehecknkxunz 95
vrbdmcoyqtnimibomigpzgpy 96
prmrbjrszzddwwywpuhbfrsv 79
mmvkajvwthgiyfoighhwopnq 82
rogjcvrvaqulmliooiosbhgj 86
puaoopoizrwfythsggrnyooj 90
wjrvnkdzwalwlhvgqbooidxq 91
mgmdvztrikpwwuxsvjrgsoni 92
wkkhnozeiztyohgpaeuyzeti 94
jvbontlmbarqwewkeuaqmhcc 96
zgpdpvmemaxnjqhkpmpnjnmy 100
rdxziwpvcdaxctrmgikccixz 110
ocublxfhrymycrpdyxraqrfl 67
jffktbhyyenkgohczjnebyfk 75
xcxvktfvakhuooumvkwdyfhy 77
lddjyhoojkrwjweavqzzkoci 80
yguwncwezawwkzrzagzvdvtk 81
fvxplzzxulajcplbjwwkyiwk 83
dgkgcdsbrxvsnfuklrutnkal 84
pxeqvjcunzpzvuuvhcrvsohp 84
lvkbmrpaxslvbzjkmbivzmww 90
xmfsaojawexkgvohwldimwhm 96
npsjcccdokuzyzqtzqgcdwjt 76
yhrvawfkyikmlwvugmgsdapy 83
gfyhqamdfbogcdvuqabqwlkr 86
dzlpkzrxqlitqnwgxsfvrxqt 93
ifgantzocqlcpropbybnxwjd 97
kzbzmdkjihrkmptxviogpikx 97
owxrsgkxixmswfsmrahjpfpm 98
akfriocrtaylakqhoeosklmc 106
ldpkbypoqkfhywjrtnampwgl 106
lpvormpatxcgsbwuuiwlblcc 110
nejckzpkyybsamdvvenlsdle 65
jztmrkuigfbhxfepkwwnqjhb 74
rtlrbqdusdsfcslslggnjdhs 76
pnobrytaxmooykzgpcqohufu 87
qrrntjimsdgvsyzqbycdygfm 87
qchhnokfczgvxzxwnomnynuq 90
fenmoknboygaxxkxvoqxcvvx 93
qhsscnseydzlzxecmltlfnzz 98
ziltlwysanfrxuunsaqsxcpv 98
rayynzxdqrtlcdxugepzcqrg 99
esfuistxkcflwrljkjaocopy 81
dyscmmucwabamvwcvcgmuohj 90
ysevfmuswcbisxztmtiepqlx 95
njghkovqbnunpufhfvckvkic 96
lqwnfonhnazyxrnlpykichlk 103
nphizskzmrwrdsmzunimmjhb 106
fzzuplbsjtwvfqzbtbbfyziv 108
dokxetijthgishceabbhtbim 110
cznqiurnoyjyrevbgcodwzog 115
elsemazqqjpudygyhoicxmgz 115
pziovueczkkgrosjthzlrvwi 83
iyobgbnvaxlxtrztcmztatet 93
yqdkzshvyyneagxkyzhoclnw 93
tqldjaqqnxxwkyhbilttvwjz 94
cehtbddlbrmhubewaqttjtml 106
fdeigeckseposhvirlaergnm 106
bmafvoqhohprnggjthkwywmq 111
fngfglbjpieslkdodrxysbus 111
ugqsecttksaundczrovirkot 112
ngzhkvbflokwodmeamckmjnv 118
kfaypxrkiesslhqulhupqvyz 79
tlnhjfpwgcrycgbjjogzujse 85
xewfefznjinlroyhssppaass 85
mkprvenwotpnnchxuqiefont 87
mbmtqapafbnxqdbribntlhvt 89
mgbgqtkcbkashtesaykngaht 94
pmuvnqgluesajtovbdgvreel 99
ywcawutljpiwgqjcucdgvtdz 105
idmlqpoezfxwlkubfkosmwrg 108
hejzzhznoctgxgkasnucgzui 111
xaweeirjzjceoppodrxkpicw 91
cunizmlqiadealwibmactsjs 94
tigvkirxgshnnvzfzfyugfxc 99
ssnnlqmhbkyslaijfbeemnor 104
fpptfflkynaifxjqfbmfftgu 106
fscqiqvaiiqwyefbpetagcll 108
jqeokmljohpuphdpqitgxkvc 109
punnwyisiqnsyhpfzyuouvtt 114
csntcrrnixbsmodawtxlcaau 115
kwakgvmkpbfrvzedwixiwalu 119
ncunguabundcthastxrgkwph 77
halongtzueyhghlxdsfkxras 87
gxqfufgltijcxqtwipbzffsc 91
hllcfwwxbfkejrjhowgwzxpz 92
rqiwaxqgwgnqntbiekkbaezl 92
tamafgcyxmvzlnfhvszilaqr 92
mmlzlagbpwhgfdkpvdkhzgpi 95
xmbwsrfztsqgylhuricxppvu 97
pmcxhmhrzstepqcsvmwgdhpr 99
qopephvekqrskuwghzkchnvc 99
ifwxpywlrfduxkqnspoyksok 79
knhbccxjfeozmoshpftjajuo 81
ppbirksexahzefkjlzkxpceg 87
smdcmbzlkvdxwzwzkjqiwjgp 87
trjsdqhtohxzeveoxlmbrvqu 87
kapmlladadlrpgvogdnkkywv 89
qwydlxwdjcfwdgjnbnmamkbz 89
gkrtdfdigqdetwkejqcxrnca 91
daluwdrgindylbjfjmzgyrws 92
ziuqnbwxdllownrwrfpilnxt 95
meuviyxhdwogqaekpmbicfvl 78
vazivehprtyjdkvnytcfjcea 78
jfulxnhbkulycovhkqabrgpd 82
xfzhaslwungptpenjcwoorcn 87
kuzanbnahixpivxotluknawe 88
snkjqkllimyofoaypvzxhvte 88
cxnkogdvfuvroygomwjfhkue 89
tybmeqqjrbvaikuuxyfvntdw 90
dzlkjfosmldlynmijziopdxa 93
hxqymubnymovtkymvzlqcqqw 97
irmkbapnqwwjayuszgsbdxoz 86
bbkugagakchyytztuqvgioai 90
dhzwbywjcafjcqxwsvgyekag 95
idcufakltawigicaagsccpcy 95
amotwidxlottqztdbbbawavr 97
coglvhodzcpcegpveogemawo 98
qqzbezlenzzjjdiutcojbjmd 100
mqfjqcajsbmyaxfnweyfhgbc 101
skwhmumauzralsrsxlmetiuk 102
grpjogksusepzuvawmcatuxx 103
ibdaxgenqxlehmvkkssnfamt 80
emhiofnzmacknkwdnelbbryv 88
hqeonncxvegaeeuvakhipqvf 88
orzwrlpwelqvvjwjiyumgwhm 89
lfnavkikqbifoxeusqnajfgg 91
mgrwpnowetmtiqqdvygbcayd 95
oitxohzpbyxialhxlafbpaoz 96
zuzfeaoxuytgnnvligmdswbd 96
khajdzvsruokhfglqwilihuq 100
crusrktzlwclegaufnymzuxm 107
ndqwoeitphqdmkqhqdhoiubz 78
flzuodunvelaydlmozzflixf 86
iutbwdxyyjsoogfaytanncey 90
vaeglboaegzvkhqcogharajh 92
beexnwebzqsvzezxpvmdrciv 93
vmlomuysmwgaxvjaobrljgpa 96
ndkpuimdwiwpmuoecxlqsrrs 101
fsyombnjbtsoxgpdooybvpdl 104
zvmuofmrksozpuaokumjblgq 105
xqjydsqlmlmlpjjpudeljllj 117
dmogwvgrnonldbawpgfxxrch 87
dysnwkalnbecflaxofhwbywz 89
dpenxbucpikngqetgtejughl 90
tpgvynmmtogjgrkdhknxhivh 90
jannqsjvtcirokcbmigtuiqk 95
ilvfrcqgtbtbnqmrsntwiysn 96
maztpdfyqddkhvqawlbmcqfu 98
oecsoqkaxqyknxtyyvpgbihy 108
nokmimvlbbyaevetdoijenem 115
mfexlvxjkacpggtbvjiiahfo 116
vhmhcysdcedpixvxgcjvyszr 82
mwvmrudsstjyutzeqjsklhku 86
qyqbicxkbkfkmcikuscgfnmo 86
qfmctrdrdujaenlmynuqikpi 87
czktjavvyqirmnjegvecofgg 91
tlnbxcmxqxxrdglpdlvsmtmj 101
belxkcuenjhkntjdtgxwoils 102
wmrsviclilcgqupxaowbxouj 102
aljxbejtlvyeobxsxgscweow 103
bwnpgbjbkvimtapygsnqrknz 106
